From: Gabriel Valcazar <gabriel.valcazar@digi.com>
Date: Thu, 7 Feb 2019 13:15:43 +0100
Subject: [PATCH] bluetooth.service: add Digi customizations

These changes include:

* Adding the bluetooth-init service as a soft dependency.
* Having the bluetooth stack start automatically on boot.

Signed-off-by: Gabriel Valcazar <gabriel.valcazar@digi.com>
---
 src/bluetooth.service.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/bluetooth.service.in b/src/bluetooth.service.in
index f799f65..d8b150c 100644
--- a/src/bluetooth.service.in
+++ b/src/bluetooth.service.in
@@ -2,6 +2,7 @@
 Description=Bluetooth service
 Documentation=man:bluetoothd(8)
 ConditionPathIsDirectory=/sys/class/bluetooth
+Wants=bluetooth-init.service
 
 [Service]
 Type=dbus
@@ -16,5 +17,5 @@ ProtectHome=true
 ProtectSystem=full
 
 [Install]
-WantedBy=bluetooth.target
+WantedBy=multi-user.target
 Alias=dbus-org.bluez.service
